sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -290,6 +301,30 @@ class wi_element<sycl::ext::oneapi::bfloat16, NumRows, NumCols, Use, Layout,
 #endif // __SYCL_DEVICE_ONLY__
 };
 
+template <typename T, size_t NumRows, size_t NumCols, use Use, layout Layout,
+          typename Group>
+inline __SYCL_ALWAYS_INLINE std::tuple<uint32_t, uint32_t>
+get_coord(wi_element<T, NumRows, NumCols, Use, Layout, Group> &we) {
+#ifdef __SYCL_DEVICE_ONLY__
+  __ocl_vec_t<uint32_t, 2> coord =
+      __spirv_JointMatrixGetElementCoordINTEL(we.M.spvm, we.idx);
+  const uint32_t row = coord[0];
+  const uint32_t col = coord[1];
+  return std::make_tuple(row, col);
+#else
+  throw runtime_error("joint matrix is not supported on host device.",
```

Comment:
Something along the lines of:
```suggestion
  throw runtime_error("get_coord is only supported on Intel XMX and AMX devices.",
```

?